modules/telemetry/telemetrymodule.h

```diff
@@ -0,0 +1,182 @@
+ * Copyright (c) 2014-2024                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#ifndef __OPENSPACE_MODULE_TELEMETRY___TELEMETRYMODULE___H__
+#define __OPENSPACE_MODULE_TELEMETRY___TELEMETRYMODULE___H__
+
+#include "openspace/util/openspacemodule.h"
+
+#include <modules/telemetry/include/telemetrybase.h>
+#include <ghoul/misc/boolean.h>
+#include <openspace/properties/optionproperty.h>
+#include <openspace/properties/scalar/boolproperty.h>
+#include <openspace/properties/scalar/intproperty.h>
+#include <openspace/properties/stringproperty.h>
+#include <atomic>
+#include <condition_variable>
+
+namespace openspace {
+
+class TelemetryBase;
+
+class TelemetryModule : public OpenSpaceModule {
+public:
+    constexpr static const char* Name = "Telemetry";
+
+    /**
+     * This mode only affects telemetries that send angle information. For example, the
+     * NodesTelemetry and the PlanetsSonification. More documentation for the elevation
+     * part of the angles is located further below.
+     *
+     * Horizontal: Calculate the angle to the object in the horizontal direction within
+     *             the camera plane. This mode is suitable for flat displays or forward
+     *             facing immersive envierments (such as the Norrköping dome theater).
```

Comment:
```suggestion
     *             facing immersive environments (such as the Norrköping dome theater).
```

Reply:
Should we really refer to the Norrköping dome here?  Could instead say ", like a uni-driectional dome theater", to make it meaningful for a larger number of people that are not familiar with our specific dome